src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,60 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book testBook1 = bookDao.create(new Book("Harry Potter and the Philosophers Stone",
```

Comment:
Creating a new Book with a title 'Harry Potter and the Philosophers Stone' might violate the checklist item that states 'Don't make `books.title` UNIQUE'. If the `books.title` column is set to UNIQUE in the database schema, this will cause an issue when creating multiple books with the same title.